core.py

```diff
@@ -140,66 +121,61 @@ def process_order(self, cust_id):
             if option == constants.CUST_OPT_ADD_FOOD_TO_ORDER:
                 food_id = input("Enter food ID: ")
                 food_qty = input("Enter food quantity: ")
-                self.customer.add_food_to_order(o.order_id, food_id, food_qty, session)
-                print("Add {}!".format("successful" if True else "not successful."))
+                a = self.customer.add_food_to_order(session, o.order_id, food_id, food_qty)
+                print("Add {}!".format("successful" if a else "not successful."))
 
             elif option == constants.CUST_OPT_REMOVE_FOOD_TO_ORDER:
                 food_id = input("Enter food ID: ")
-                self.customer.remove_food_to_order(o.order_id, food_id, session)
-                print("Items {}".format("removed" if True else "not removed"))
+                r = self.customer.remove_food_to_order(session, o.order_id, food_id)
+                print("Items {}".format("removed" if r else "not removed"))
 
             elif option == constants.CUST_OPT_UPDATE_FOOD_TO_ORDER:
                 food_id = input("Enter food ID: ")
                 food_qty = input("Enter food quantity: ")
-                self.customer.update_food_to_order(o.order_id, food_id, food_qty, session)
-                print("Items {}".format("updated" if True else "not updated"))
+                u = self.customer.update_food_to_order(session, o.order_id, food_id, food_qty)
+                print("Items {}".format("updated" if u else "not updated"))
 
             option = int(input(selection))
         
         if o:
-            print("Order {}. \nOrder number: {}".format("generated", o.order_id if True else "not generated"))
-
+            print("Order {}. \nOrder number: {}".format("generated", o.order_id if o else "not generated"))
```

Comment:
Don't check for `o` here, just print it. Check for `o` on top. On top, do like this:

```python
if not o:
  return
```